src/app/organizations/create-organization/create-organization.component.html

```diff
@@ -47,13 +47,17 @@ <h2 class="left__upload__subtitle">Add logo</h2>
                           placeholder="Enter description"
                           id="group-description">
                 </textarea>
+
+                <div class="test">
```

Comment:
Is there a reason why you named this class 'test' ?

Reply:
I probably left this class on accident, thanks